### ISSUE TITLE
Elevation sensors are not working

### ISSUE THREAD

**Issue by Mopfus** [pnbruckner/ha-sun2#77]
For some reason I don't understand, the elevation sensors are not working for me.

I added the following to the `configuration.yaml`:
```
  - platform: sun2
    monitored_conditions:
      - sunrise
      - sunset
      - sun_phase
      - daylight
      - night
      - elevation
      - min_elevation
      - max_elevation
```

But I only see the following sensors:
- binary_sensor.above_horizon
- sensor.civil_daylight
- sensor.daylight
- sensor.night
- sensor.sun_phase
- sensor.sunrise
- sensor.sunset

See also here:
![2023-03-11 13 58 57 192 168 0 156 b11b807e66a7](https://user-images.githubusercontent.com/80747454/224487800-9d0a1b39-61ce-432c-af91-e61a69bb9579.png)

Home Assistant 2023.3.3
Supervisor 2023.03.1
Operating System 9.5
Frontend 20230309.0 - latest

**Comment by pnbruckner** [comment]
But you have `sensor.civil_daylight` and that's _not_ in your config. Are you sure you saved your changes to configuraiton.yaml and then restarted HA? Do you see any error messages?

**Comment by Mopfus** [comment]
Thank you, that you mentioned that. I was restarting several times (even the host system). Because of your comment I looked again closely at the `configuratin.yaml`. And I found out, that somehow my **Studio Code Server** AddOn gave me this error message. And I have overseen that...
![2023-03-12 18 42 27 192 168 0 156 57806edb73c2](https://user-images.githubusercontent.com/80747454/224565251-409d2efa-7d9a-4346-a9e5-cca5d0119f08.png)


Now everything works like a charm! Thank you very much.
